spec/system/jobseekers_can_unlock_their_account_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+RSpec.describe "Jobseekers can unlock their account" do
+  let(:jobseeker) { create(:jobseeker) }
+
+  before do
+    allow(JobseekerAccountsFeature).to receive(:enabled?).and_return(true)
+    visit root_path
+    within("nav") do
+      click_on I18n.t("buttons.sign_in")
+    end
+  end
+
+  context "when the jobseeker has one sign-in attempt remaining" do
+    before { jobseeker.update!(failed_attempts: Devise.maximum_attempts) }
```

Comment:
This things make me a bit torn. 

One the one hand, one line is nice and clean and tidy
But on the other hand, I like consistency and the block notation is super clear

Just commenting so we think about it really

Reply:
I like using the one-liners where possible. Decreasing the vertical space between different specs makes it that bit easier to quickly compare them. However, there are also readability costs with using inconsistent code style.